bundles/org.openhab.binding.ojelectronics/src/main/java/org/openhab/binding/ojelectronics/internal/ThermostatHandler.java

```diff
@@ -158,6 +254,42 @@ private void updateGroupName(Thermostat thermostat) {
         updateState(BindingConstants.CHANNEL_OWD5_GROUPNAME, StringType.valueOf(thermostat.groupName));
     }
 
+    private void updateVacationEnabled(Thermostat thermostat) {
+        updateState(BindingConstants.CHANNEL_OWD5_VACATIONENABLED,
+                thermostat.online ? OpenClosedType.OPEN : OpenClosedType.CLOSED);
+    }
+
+    private void updateVacationBeginDay(Thermostat thermostat) {
+        updateState(BindingConstants.CHANNEL_OWD5_VACATIONBEGINDAY,
+                new DateTimeType(
+                        ZonedDateTime.ofInstant(thermostat.vacationBeginDay.toInstant(), ZoneId.systemDefault())
+                                .truncatedTo(ChronoUnit.DAYS)));
```

Comment:
Instead of using the default ZoneId, try using the one provided by the TimeZoneProvider osgi service. You will have to get the service as a `@Reference` in the ThingHandlerFactory and then pass it in through the constructor of this handler.

Reply:
Thanks for the hint. I fixed it.